cryptoconditions/types/ed25519.py

```diff
@@ -66,7 +66,7 @@ def sign(self, message, private_key):
         #   .update(Buffer.concat([this.messagePrefix, this.message]))
         #   .digest()
 
-        self.signature = sk.sign(message, encoding=None)
+        self.signature = sk.sign(message, encoding='bytes')
```

Comment:
This didn't change anything. `encoding='bytes'` is that same as the old `to_bytes()`